projects/step-core/src/lib/modules/dynamic-forms/components/dynamic-textfield/dynamic-textfield.component.ts

```diff
@@ -54,6 +54,13 @@ export class DynamicTextfieldComponent implements ControlValueAccessor {
     this.emitChanges();
   }
 
+  preventNumberInvalidChars(event: KeyboardEvent): void {
+    const numberInputInvalidCharts = ['+', '-', 'e'];
```

Comment:
```numberInputInvalidCharts``` => ```numberInputInvalidChars```

Reply:
also maybe worth for this to be extracted in a directive that would be applied on an input and provided with forbidden symbols